src/world/World.php

```diff
@@ -899,8 +900,12 @@ public function sendTime(Player ...$targets) : void{
 		if(count($targets) === 0){
 			$targets = $this->players;
 		}
+
 		foreach($targets as $player){
-			$player->getNetworkSession()->syncWorldTime($this->time);
+			$ev = new WorldSyncTimeEvent($this, $player, $this->time);
```

Comment:
It's preferred to hold an array of all recipients instead of spamming out event calls to all handlers

Reply:
The original way was preferable